core/pkg/launch/wandb_config.go

```diff
@@ -0,0 +1,60 @@
+package launch
+
+import (
+	"github.com/wandb/wandb/core/internal/data_types"
+	"github.com/wandb/wandb/core/pkg/service"
+)
+
+// Represents the wandb config filters received by the internal process.
```

Comment:
This is also a bit opaque. What are wandb config filters? The doc should help a reader understand why this exists.